src/components/Issues.js

```diff
@@ -60,21 +60,26 @@ function Issues({repoName, owner}) {
   return owner ? (
     totalCount > 0 ? (
       <Card fitted>
-        <CardPadding>
+        <CardHeader style={{paddingLeft: 45}}>
```

Comment:
Let's not do inline styles here, and try to adhere to the 8px spacing in the https://github.com/bdougie/open-sauced/blob/master/src/styles/variables.js

Reason for the 8pt style guide is the dashboard stay consistent for all screens and stay manageable

https://medium.com/swlh/the-comprehensive-8pt-grid-guide-aa16ff402179